### PR TITLE
Issue 7: Delete Image

### PR DESCRIPTION
**Pull request by kkirasich**

### CODE REVIEW

File: app/views/images/show.html.erb [hunk 1]
```diff
@@ -2,4 +2,6 @@
   <p class="js-tag"><%= tag %></p>
 <% end %>
 
+<%= link_to 'Delete', image_path(@image), method: :delete, data: {confirm: 'Are you sure?' } %>
```
Comment (darreneng) [inline app/views/images/show.html.erb]: this is a nice helper, but it's going to generate an `a` tag here - for accessibility reasons it's usually best to only use `a` tags for links to pages. We can use bootstrap css classes to make this look like a link if we need to though.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#Accessibility_concerns

File: app/controllers/images_controller.rb [hunk 2]
```diff
@@ -24,4 +24,11 @@ def create
   def show
     @image = Image.find(params[:id])
   end
+
+  def destroy
+    @image = Image.find(params[:id])
```
Comment (darreneng) [inline app/controllers/images_controller.rb]: what happens if the image isn't present?

ie. what happens if the user has a stale tab and tries to delete an image that has already been deleted?

**Comment by kkirasich** [comment]
[Issue 7: Delete Image](https://trello.com/c/dFqOuCPC/31-issue-7-delete-image)

File: app/views/images/index.html.erb [hunk 3]
```diff
@@ -7,5 +7,5 @@
 
 <br>
 <% @images.each do |img| %>
-  <%= image_tag(img.url, class: 'image') %>
+  <%= link_to image_tag(img.url, class: 'image'), image_path(id: img.id) %>
```
Comment (darreneng) [inline app/views/images/index.html.erb]: looks like this change is unrelated to the delete functionality. Should this be in a separate commit?

Also this is minor but we can also pass the image or id as an argument to the `image_path` helper (ie. not a keyword arg), e.g. `image_path(img)`
